utils/RunONNXModel.py

```diff
@@ -158,6 +160,14 @@ def check_non_negative(argname, value):
     help="Path to a folder containing reference inputs and outputs stored in protobuf."
     " If --verify=ref, inputs and outputs are reference data for verification",
 )
+data_group.add_argument(
+    "--load-ref-from-numpy",
+    metavar="PATH",
+    type=str,
+    help="Path to a python script that defines variables inputs and outputs that are a list of numpy arrays. "
```

Comment:
Great, please add that this script expect 2 lists, named 'inputs' an 'outputs' which lists numpy arrays.
Is outputs optional?
Please add a bit of details.